### ISSUE TITLE
Live regions being announced twice by NVDA on Windows

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#5896]
### System details

    RStudio Edition : Server
    RStudio Version : 1.3.675
    OS Version      : Windows-10
    R Version       : 3.6.1

### Steps to reproduce the problem
- start NVDA, load RStudio Server in Chrome or FireFox
- ensure screen reader support enabled in Tools/Global/Options/Accessibility
- Tools / Addins / Browse Addins
- Type a couple letters in the filter field to narrow the results, wait a couple seconds

### Describe the problem in detail
- Announcement is made twice, e.g. "Found 3 addins matching foo"

### Describe the behavior you expected
- Only announce once
- Possibly due to work done here causing live region to be updated both in main window and dialog: #5893
- Otherwise, take a look here for similar problems elsewhere: https://github.com/nvaccess/nvda/issues/7996
- Does not repro on macOS with VoiceOver using Chrome or Safari (which is why I didn't encounter it during development)

Cannot repro on Windows Desktop IDE due to https://github.com/rstudio/rstudio/issues/5895
